substrate/frame/examples/multi-block-migrations/src/migrations/v1/mod.rs

```diff
@@ -0,0 +1,114 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! # Multi-Block Migration v1
+//!
+//! This module showcases a simple migration that iterates over the values in the
+//! [`old::MyMap`](`crate::migrations::v1::old::MyMap`) storage map, transforms them,
+//! and inserts them into the [`MyMap`](`crate::pallet::MyMap`) storage map.
+
+use super::PALLET_MIGRATIONS_ID;
+use crate::pallet::{Config, MyMap};
+use frame_support::{
+	migrations::{MigrationId, SteppedMigration, SteppedMigrationError},
+	pallet_prelude::PhantomData,
+	weights::WeightMeter,
+};
+
+mod benchmarks;
+mod tests;
+pub mod weights;
+
+/// Module containing the OLD storage items.
+///
+/// Before running this migration, the storage alias defined here represents the
+/// `on_chain` storage.
+// This module is public only for the purposes of linking it in the documentation. It is not
+// intended to be used by any other code.
+pub mod old {
```

Comment:
I think we settled on prefering the old version for the mod here rather than old. but not super fussed. 
```suggestion
pub mod v0 {
```